### DISCUSSION TITLE
Star trek games

### DISCUSSION THREAD

**Discussion by JHstrutton2000** [JACoders/OpenJK#1152] [General]
How hard would it be to get the old star trek games (elite force 1) running on your engine? Just wondering. I know there both running the idtech 3 engine.

**Comment by ensiform** [comment]
EF1 MP holomatch already exists in patch form for ioquake3 as it has less in common with this engine. @zturtleman may have an updated version of it I'm not positive though.

SP codebase isn't available at all to go off of.

**Comment by zturtleman** [comment]
I maintain a EF1 multiplayer engine as [Lilium Voyager](https://clover.moe/lilium-voyager/). There is also [ioEF-cMod](https://stvef.org/cmod). (The original EF1 MP reimplementation: [ioEF](http://thilo.tjps.eu/efport-progress/), source patches applied to ioquake3: [ioef-archive](https://github.com/zturtleman/ioef-archive).)

I would recommend the [Star Trek: Elite Force discord](https://discord.gg/hqSmErp) for Elite Force discussion.

**Reply by JHstrutton2000** [reply]
Oh wow thank you, I've always wondered if there were people working on this.
